app/services/unsubscribe_all_service.rb

```diff
@@ -7,6 +7,10 @@ def initialize(subscriber, reason, **)
   end
 
   def call
-    UnsubscribeService.call(subscriber, subscriber.active_subscriptions, reason)
+    ended_count = subscriber.active_subscriptions
```

Comment:
Minor: this surprised me, as it doesn't seem to be documented behaviour.

Reply:
I'm confused as to what you're referring to, could you explain?

Reply:
Oops, sorry - I can see it's a bit vague. The docs for `update_all` don't mention it returns the count of updated records (unless I missed that part). Since that's the actual behaviour, I think it's fine as-is (just a bit surprising).

Reply:
Oh yes I see, not like rails to have incomplete docs! I assumed it was similar behaviour to `delete_all` which returns the amount of rows effected. 

There is a test at least which verifies the return of `update_all` so as you say, should be okay! 👍

Reply:
Looks like the documentation for it is relatively new: https://github.com/rails/rails/commit/b488d8e08950685ae978b118af813516ce71fecc

I wondered if it depended on the RDBMs but it seems to be universal.